dot.js

```diff
@@ -0,0 +1,23 @@
+let clicked=false;
+
+function myFunc(){
```

Comment:
Let's wrap all js code inside an `IIFE` `(() => {})()`

Reply:
The `clicked` variable's name is confusing, a better name would be `isMenuOpen` also having the variable inside the `IIFE` would make it a local variable instead of a global one.